packages/react-native/src/primitives/Radio/getRadioDimensions.ts

```diff
@@ -16,23 +16,31 @@ import { RadioDimensions, RadioStyles, Size } from './types';
  * (e.g., 'small' => { radioContainerSmall, radioDotSmall } )
  */
 
+export const RADIO_DOT_PROPORTION = 0.6;
```

Comment:
Nice! Conjugate golden ratio.  Would be nice to add a comment on why we landed on this value or purpose to help future readers.

Reply:
The Silver Ratio! :)